src/stats.rs

```diff
@@ -0,0 +1,53 @@
+use anyhow::Result;
+use itertools::Itertools;
+use osmpbfreader::OsmId;
+use osmpbfreader::OsmObj;
+use std::collections::BTreeMap;
+use std::collections::HashMap;
+use std::io;
+use std::vec::IntoIter;
+
+use crate::util::filter_all_relations;
+
+fn sort_count(map: HashMap<&str, usize>) -> IntoIter<(&str, usize)> {
```

Comment:
API feedback, [`IntoIter`](https://doc.rust-lang.org/std/iter/trait.IntoIterator.html) ist der konkrete Typ, den deine impl zurückgibt, aber das ist für caller nicht wirklich relevant. Was für die caller relevant ist, ist, dass dein Rückgabetyp in einen [`Iterator`](https://doc.rust-lang.org/std/iter/trait.Iterator.html) konvertieren kann (damit kannst du dann darüber iterieren).

```suggestion
fn sort_count(map: HashMap<&str, usize>) -> impl Iterator<Item = (&str, usize)> {
```